scripts/plotting/tem.py

```diff
@@ -150,7 +162,6 @@ def tem(adf):
             # Check redo_plot. If set to True: remove old plot, if it already exists:
             if (not redo_plot) and plot_name.is_file():
                 #Add already-existing plot to website (if enabled):
-                adf.debug_log(f"'{plot_name}' exists and clobber is false.")
```

Comment:
Is there a reason you removed this `debug_log` statement?

Reply:
I probably accidentally over-rid this line, thanks